enterprise/server/remote_execution/containers/docker/docker.go

```diff
@@ -158,6 +159,8 @@ func (r *dockerCommandContainer) Run(ctx context.Context, command *repb.Command,
 		}()
 	}()
 
+	ctx, cancel := background.ExtendContextForFinalization(ctx, containerFinalizationTimeout)
```

Comment:
I think this will effectively allow the task forever, since `copyContainerLogs` blocks until the task is complete.

Maybe the logged errors are WAI? (or we could silence them)

Reply:
more context: `ContainerStart` (above) starts the task, but doesn't wait for it to finish

Reply:
err... it doesn't allow tasks to run forever, but rather restricts all tasks to the 10s timeout

Reply:
err... actually ExtendContextForFinalization will only add the 10s if there is less than 10s remaining -- so I think in most cases, extending the context here might not change the timeout

Reply:
Ah, so it would extend all execution deadlines by 10s?

I'm seeing some really large test logs (1.5GB+) and seeing some funky behavior. Thought this might come into play since I was seeing these in the logs but I might be wrong.

Reply:
Just saw your follow up - that makes sense. Back to the drawing board lol.